### PR TITLE
[Statistics] Rename link in Behavioral statistics tab for clarity

### LINKED ISSUE #5087: [Statistics] Rename link in Behavioral statistics tab for clarity
Please rename the link on the Behavioral Statistics tab of the Statistics module for clarity (see screenshot)

Current name: Click here for breakdown per participant

New name: Per Instrument Stats - All Sites

Statistics_Bx-Tab_Rename-Link_2018-02-14

https://redmine.cbrain.mcgill.ca/issues/13919

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes
This renames the link in Behavioral statistics tab.

Haowei Qiu, GOSC Applicant
CC @christinerogers 

#### Link(s) to related issue(s)

* Resolves #5087  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: modules/statistics/templates/form_stats_behavioural.tpl [hunk 1]
```diff
@@ -107,7 +107,7 @@
         </tbody>
     </table>
     <div>
-        <b><a href='{$baseurl}/statistics/statistics_dd_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Click here for breakdown per participant{if $CurrentSite} for {$CurrentSite.Name} {/if} {if $CurrentProject} for {$CurrentProject.Name} {/if}</a></b>
+        <b><a href='{$baseurl}/statistics/statistics_dd_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Per Instrument Stats - All Sites{if $CurrentSite} for {$CurrentSite.Name} {/if} {if $CurrentProject} for {$CurrentProject.Name} {/if}</a></b>
```
Comment (driusan) [inline modules/statistics/templates/form_stats_behavioural.tpl:110]: Won't this change the label to "Per Instrument Stats - All Sites for $SiteName"?

Comment (haoweiqiu) [reply modules/statistics/templates/form_stats_behavioural.tpl]: @driusan Thank you very much for your comment! I just modified the link.

**Comment by christinerogers** [comment]
Hi @haoweiqiu - could you rebase this onto main, resolve any conflicts, and then re-request my review?  thanks!

**Comment by haoweiqiu** [comment]
@christinerogers I have updated this branch, however, it seems that I do not have access to add code reviewers. Thanks!

**Comment by racostas** [comment]
Hi @haoweiqiu, 

I was requested to review this PR. The changes are looking good to me. I would said it is the same situation for the link associated to the `Double Data Entry Statistics` below. I would suggest to change this label too in this case.

Maybe something like:
**Per Instrument Data Entry Stats - All Sites** (for link 1)
**Per Instrument Double Data Entry Stats - All Sites** (for link 2)

I would like to heard the opinion of @driusan and maybe @christinerogers or @ridz1208 in any case.

File: modules/statistics/templates/form_stats_behavioural.tpl [hunk 2]
```diff
@@ -55,7 +55,7 @@
 
 
 
-    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Click here for breakdown per participant {if $CurrentSite} for {$CurrentSite.Name} {/if} {if $CurrentProject} for {$CurrentProject.Name} {/if}</a></b>
+    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Per Instrument Stats - All Sites</a></b>
```
Comment (christinerogers) [inline modules/statistics/templates/form_stats_behavioural.tpl:58]:
```suggestion
    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Per Instrument Data Entry Stats - All Sites</a></b>
```

Comment (christinerogers) [reply modules/statistics/templates/form_stats_behavioural.tpl:58]: Agreeing with @racostas ' review : here's the suggestion/change requested.

**Comment by haoweiqiu** [comment]
@christinerogers Sure! I'll look into these right now!

File: modules/statistics/templates/form_stats_behavioural.tpl (hunk 2 above)
Comment (haoweiqiu) [reply modules/statistics/templates/form_stats_behavioural.tpl]: @christinerogers I have resolved the mentioned issues!

**Comment by haoweiqiu** [comment]
@christinerogers I have resolved the mentioned issues. Thanks!

File: modules/statistics/templates/form_stats_behavioural.tpl [hunk 3]
```diff
@@ -97,7 +97,7 @@
             {/foreach}
         </tr>
         <tr>
-            <td class=" pis">Per Instrument Stats</td>
+            <td class=" pis">Per Instrument Data Entry Stats</td>
```
Comment (christinerogers) [inline modules/statistics/templates/form_stats_behavioural.tpl:100]:
```suggestion
            <td class=" pis">Per Instrument Double Data Entry Stats</td>
```
ok, but needs to say *Double* Data Entry stats

Comment (christinerogers) [reply modules/statistics/templates/form_stats_behavioural.tpl]: hang on, is this really the All-Sites link ?  I'll have to go check an instance...

Comment (ridz1208) [reply modules/statistics/templates/form_stats_behavioural.tpl:100]: @christinerogers it never adds a site id to the query (only the links above in the table do so) but it still appends a project when a project is selected

Comment (ridz1208) [reply modules/statistics/templates/form_stats_behavioural.tpl:100]: no its not an all sites. it's only the site above in the header

**Comment by christinerogers** [comment]
Ok let's close this PR for now, seems like we need to revisit what this link should be doing from the old ticket #5087.  
Thanks for your efforts here @haoweiqiu !  (We'll pick this back up in a few weeks when possible. )

**Pull request closed without merging**